src/components/Post/GovernanceSideBar/Referenda/VoteReferendum.tsx

```diff
@@ -530,19 +530,34 @@ const VoteReferendum = ({ className, referendumId, onAccountChange, lastVote, se
 		};
 		if (!voteTx) return;
 
-		await executeTx({
-			address,
-			api,
-			apiReady,
-			errorMessageFallback: 'Transaction failed.',
-			network,
-			onFailed,
-			onSuccess,
-			params: network === 'equilibrium' ? { nonce: -1 } : {},
-			proxyAddress: selectedProxyAddress,
-			setStatus: (status: string) => setLoadingStatus({ isLoading: true, message: status }),
-			tx: voteTx
-		});
+		if (showProxyDropdown) {

```

Comment:
this can be done in one line
proxyAddress: showProxyDropdown && selectedProxyAddress ? selectedProxyAddress : ''.

A better implementation is :
As soon as the user de-selects the proxy checkbox selectedProxyAddress should be emptied